robyn/env_populator.py

```diff
@@ -26,11 +24,11 @@ def parser(config_path=CONFIG_PATH):
 
 
 # check for the environment variables set in cli and if not set them
-def load_vars(variables = None):
+def load_vars(variables = None, root = None):
```

Comment:
Two last nits: let's call the variable as `project_root` only as we want the `**kwargs` to be related to the function name,

Second, can you please use the `pre-commit` hooks like mentioned in here https://github.com/sansyrox/robyn#%EF%B8%8F-to-develop-locally

Thanks again :smile: